options/posix/include/net/if.h

```diff
@@ -47,6 +48,22 @@ struct ifreq {
 		char *ifru_data;
 	} ifr_ifru;
 };
+#define ifr_name ifr_ifrn.ifrn_name
```

Comment:
These defines are duplicate, see right below.

Reply:
Dropped the offending commit.